README.md

```diff
@@ -30,14 +30,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+DOM, or document object model is a programming APi for HTML and XML document. It defines the logical structure of documents and the way the document is accessed and manipulated
```

Comment:
perhaps a more tangible definition is that the DOM is an object representation of the HTML